### ISSUE TITLE
3.0b1 - Invalid Function Reference (SendToASU ->SendToASULog)

### ISSUE THREAD

**Issue by iDrewbs** [Macjutsu/super#32]
There are two lines of code in version 3.0b1 producing an error that reference a non-existing function (SendToASU). I believe these should be referencing the function SendToASULog:

```
Line 2995: sendToASU "Status: Installing Recommended Software Update $((i + 1)): ${recommendedLABLES[i]}..."
Line 3020: sendToASU "Status: Installed Recommended Software Update: $updateRECOMMENDED."
```

**Comment by Macjutsu** [comment]
Weird, I'll check that out later today and submit a new build if I need to, considering that should be a pretty simple fix.

**Comment by Macjutsu** [comment]
Fixed in [`super` v3.0b2](https://github.com/Macjutsu/super/releases/tag/v3.0-b2).
